pkg/agent/syssetup/setup.go

```diff
@@ -32,6 +32,7 @@ func Configure() {
 	loadKernelModule("nf_conntrack")
 	loadKernelModule("br_netfilter")
 	loadKernelModule("iptable_nat")
+	loadKernelModule("ip6table_nat")
```

Comment:
This seems to assume dual-stack and hence might incur unwanted side-effects on the host (such as other ipv6 modules getting loaded).

Reply:
Reworked to make IPv6 configuration conditional